README.md

```diff
@@ -281,8 +281,10 @@ When payments are scheduled to be sent out, your Filecoin wallet will receive a
 
 ### Node monitoring
 
-- https://dashboard.saturn.tech - View historical data on your bandwidth contributions, FIL earnings, and more.
-- https://orchestrator.strn.pl/stats - View detailed, real-time stats on every Saturn node.
+- https://dashboard.saturn.tech - View detailed, real-time stats on every Saturn node.
+- https://dashboard.saturn.tech/address - View historical data on your bandwidth contributions, FIL earnings, and more.
+- https://explorer.saturn.tech - View a 3D geospatial visualization of the Saturn network, along with nodes and network statistics.
+- https://github.com/31z4/saturn-moonlet - Self-hosted Saturn monitoring using Prometheus and Grafana. View detailed, real-time and historical data on your nodes and earnings, setup alerts, and more.
```

Comment:
```suggestion

### Community Tools

These Saturn tools are maintained by community members outside the Saturn core team.

- https://github.com/31z4/saturn-moonlet - Self-hosted Saturn monitoring using Prometheus and Grafana. View detailed, real-time and historical data on your nodes and earnings, setup alerts, and more.
```

@DiegoRBaquero does this resolve your concern. I'm fine with the change if it incorporates the disclaimer, but I don't want to override you.

@31z4,
Hey I'm @hannahhoward the (new-ish) Saturn tech lead. I am personally fine to merge with the above change to make it clear this is not a core team developed tool, if it resolves @DiegoRBaquero 's concern. My only other concern is -- are you actively maintaining and developing this tool? Do you think we can safely say that in six months we'll still be running it?

Reply:
Hi @hannahhoward 👋 Yes, I'm going to continue maintain Saturn Moonlet and willing to build community around it.